projects/plugins/jetpack/extensions/blocks/podcast-player/providers.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
It can be a bit confusing to have both `providers.js` file and `providers` subdirectory in one directory. Perhaps this could go into `index.js` of `providers` subdirectory?

Reply:
That's a good idea actually. :)

Reply:
That's a good idea actually. :)